example/provider-local/garden/base/cloudprofile.yaml

```diff
@@ -27,6 +27,8 @@ spec:
     - version: 1.0.0
       cri:
       - name: containerd
+        containerRuntimes:
+          - type: gvisor
```

Comment:
Please remove this from this file - we don't install the gvisor extension by default.